script.js

```diff
@@ -0,0 +1,48 @@
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+const buttonAdd = document.getElementById('add');
+let booksData = [];
+const check = localStorage.getItem('books');
+if (check != null) {
+  booksData = JSON.parse(check);
+}
+
+buttonAdd.addEventListener('click', () => {
+  const bookTitle = title.value;
+  const bookAuthor = author.value;
+  const bookData = [bookTitle, bookAuthor];
+  booksData.push(bookData);
+  const allData = JSON.stringify(booksData);
+  localStorage.setItem('books', allData);
+  // eslint-disable-next-line no-restricted-globals
+  location.reload();
+});
+
+let getBooks = localStorage.getItem('books');
+getBooks = JSON.parse(getBooks);
+
+if (getBooks.length > 0) {
+  const booksSection = document.getElementById('books');
+  let book = '';
+  for (let i = 0; i < getBooks.length; i += 1) {
+    book += `<div id="book${i}"><p>${getBooks[i][0]}</p>
+    <p>${getBooks[i][1]}</p>
+    <button class="button" value=${i}>Remove</button><br>
+  <hr>
+  </div>`;
+  }
+  booksSection.innerHTML = `${book}`;
+}
+
+const deleteBtn = document.querySelectorAll('.button');
+
+deleteBtn.forEach((deleteBtn) => {
+  deleteBtn.addEventListener('click', () => {
+    const index = deleteBtn.value;
+    getBooks.splice(index, 1);
+    getBooks = JSON.stringify(getBooks);
+    localStorage.setItem('books', getBooks);
```

Comment:
- Same here kindly make sure that you are calling a function to remove books. This is a project requirement. :pray: :pray: 
![image](https://user-images.githubusercontent.com/98363075/204406064-baeb8684-2163-45ed-8587-7890d98f8b76.png)

Reply:
![Captura desde 2022-11-28 19-40-13](https://user-images.githubusercontent.com/113067895/204417392-84d394b7-4b37-4086-8b68-e7dd7f70ed4a.png)
@Santiago220991 sorry, but I actually created the function. Could you check again, please? This is the last commit a830103

Reply:
Yes, I know and that is great the only issue right now is your README file. Please use the new template provided by Microverse. :+1: :smiley: :smiley: